README.md

```diff
@@ -14,10 +14,10 @@ _nichtparasoup_ is a hackspaces entertainment system.
 It randomly displays images from
 [instagram](https://instagram.com),
 [pr0gramm](https://pr0gramm.com) and
-[reddit](https://reddit.com).  
```

Comment:
could you revert the removal of the trailing spaces.
they were intended for a line break